lib/3scale/backend/server/falcon.rb

```diff
@@ -0,0 +1,48 @@
+module ThreeScale
+  module Backend
+    module Server
+      class Falcon
+        extend ThreeScale::Backend::Server::Utils
+
+        def self.start(global_options, options, args)
+          # Falcon does not support:
+          # - options[:daemonize]
+          # - options[:logfile]
+          # - options[:errorfile]
+          # - options[:pidfile]
+
+          manifest = global_options[:manifest]
+          return unless manifest
+
+          argv = ['falcon']
+          argv_add argv, true, '--bind', 'http://0.0.0.0'
+          argv_add argv, options[:port], '--port', options[:port]
+
+          server_model = manifest[:server_model]
+          argv_add argv, true, '--count', server_model[:workers].to_s
+        end
+
+        def self.restart(global_options, options, args)
+          argv = ['falcon', 'supervisor']
+          argv_add argv, true, 'restart'
+        end
+
+        def self.stop(global_options, options, args)
+          puts 'Not implemented'
```

Comment:
Have you considered using `STDERR`? Or even `raise`? Also, there should be some means to stop the server gracefully without `R`esetting TCP connections (even if it is just sending a SIGTERM?).

Reply:
ping @davidor

Reply:
I'll change this to raise instead :+1:

Reply:
Done